traffic-sniffer/lib/request-metadata/sniffer-manager/sniffer-manager.ts

```diff
@@ -1,11 +1,16 @@
 import { PathResponseData } from "../../../types/types";
 import { Sniffer, SnifferConfig } from "../sniffer/sniffer";
+import { SetupFile } from "../setup-file"
 
 export class SnifferManager {
   private readonly sniffers: Sniffer[];
+  private setupFileData: SetupFile;
 
   constructor() {
     this.sniffers = [];
+    
+    this.setupFileData = new SetupFile();
+    this.loadSnifferFromSetupFile();
```

Comment:
Create ConfigLoader class so we could change it to database loader in the furute